src/vs/workbench/contrib/chat/browser/chat.contribution.ts

```diff
@@ -130,6 +130,11 @@ configurationRegistry.registerConfiguration({
 			description: nls.localize('chat.experimental.detectParticipant.enabled', "Enables chat participant autodetection for panel chat."),
 			default: null
 		},
+		'chat.experimental.imageAttachments': {
```

Comment:
I'm not sure the setting should be a registered user-visible setting, because it won't work with current models if they enable it.